uniffi.toml

```diff
@@ -1,4 +1,4 @@
 [bindings.kotlin]
 package_name = "com.spruceid.mobile.sdk.rs"
 # Disabling Android optimisations due to uniffi tests breaking on missing import android
-android = false
+android = true
```

Comment:
Based on that comment, I think this needs to be false.

Reply:
Ah, I forgot to remove the comment. I don't believe this is correct anymore. See https://github.com/mozilla/uniffi-rs/pull/1997 and https://github.com/mozilla/uniffi-rs/issues/1996 which I believe fixed this.